lib/util/jshint-reporter.js

```diff
@@ -87,24 +87,26 @@ var yargsOptionDefiniton = {
     string: true
   }
 };
-var checkJsHintReporter = yargs.createCheck()
-  .withGate(function (argv) {
-    return !argv.help;
-  })
-  .withTest({
-    reporter: function(value) {
-      if (typeof value === 'undefined') {
-        return;
-      }
+function checkJsHintReporter(argv) {
```

Comment:
since yargs is now dry, yargs options should now be decoupled from this implementation file (and promoted to the task file)